src/pages/FriendFind/index.tsx

```
@@ -0,0 +1,68 @@
+        className="h-12"
+        value={value}
+        onChange={onChange}
+      />
+
+      {isLoading && <UserCardSkeleton count={3} className="space-y-3 mt-6" />}
+
+      {searchResults.length > 0 ? (
+        <ul className="space-y-3 mt-6">
+          {searchResults.map((user) => (
+            <UserCard name={user.nickname} message={user.statusMessage} asChild>
+              <li>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)